src/__tests__/__snapshots__/index.js.snap

```diff
@@ -1,39 +0,0 @@
-// Jest Snapshot v1, https://goo.gl/fbAQLP
-
-exports[`assert throws if code is unchanged + snapshot enabled 1`] = `"Code was unmodified but attempted to take a snapshot. If the code should not be modified, set \`snapshot: false\`"`;
-
-exports[`assert throws if snapshot and output are both provided 1`] = `"\`output\` cannot be provided with \`snapshot: true\`"`;
-
-exports[`assert throws if the plugin name is not inferable 1`] = `"The \`pluginName\` must be inferable or provided."`;
-
-exports[`assert throws if there's no code 1`] = `"A string or object with a \`code\` or \`fixture\` property must be provided"`;
-
-exports[`can fail tests in fixtures at an absolute path 1`] = `"actual output does not match output.js"`;
-
-exports[`default will throw if output changes 1`] = `"Expected output to not change, but it did"`;
-
-exports[`plugin is required 1`] = `"plugin is a required parameter."`;
-
-exports[`snapshot option can be derived from the root config 1`] = `"\`output\` cannot be provided with \`snapshot: true\`"`;
-
-exports[`takes a snapshot: 1. captains-log 1`] = `
-"
-var hi = \\"hey\\";
-
-      ↓ ↓ ↓ ↓ ↓ ↓
-
-var ih = \\"hey\\";
-"
```

Comment:
@necolas, we solve the double quotes problem with the un-string snapshot serializer thing.

Notice in the before snapshot, jest wraps the output in a string (because that's what it is serializing). If you checkout the serialized value in the new version of this snapshot, you'll notice there's no quotes and it doesn't have to do the weird escaping stuff.